x-pack/plugin/ml/src/main/java/org/elasticsearch/xpack/ml/extractor/GeoPointField.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+package org.elasticsearch.xpack.ml.extractor;
+
+import org.elasticsearch.search.SearchHit;
+
+import java.util.Arrays;
+import java.util.Collections;
+
+public class GeoPointField extends DocValueField {
+
+    static final String TYPE = "geo_point";
+
+    public GeoPointField(String name) {
+        super(name, Collections.singleton(TYPE));
```

Comment:
#nit  
this `Collections.singleton(TYPE)` and the others that are simply passed up to `super` could be `private static final`so that a new HashSet isn't built on every `ctor` call